core/controllers/admin.py

```diff
@@ -848,3 +848,16 @@ def post(self):
             self.render_json({})
         else:
             raise self.InvalidInputException('This app cannot send emails.')
+
+
+class InteractionsByExplorationId(base.BaseHandler):
+    """Handler to show a list of interaction id's for a exploration id."""
```

Comment:
... retrive **the** list of interactions used in **an** exploration.

Reply:
updated

Reply:
I think it's not updated correctly, can you please check?

Reply:
@aggarwalpulkit596, is this resolved?

Reply:
@aggarwalpulkit596, please reply to the last comment!

Reply:
done

Reply:
@aggarwalpulkit596 This is still not resolved?

Reply:
I guess there is some bug with the github I've changed this file long back and 
<img width="617" alt="Screenshot 2020-05-11 at 5 57 24 PM" src="https://user-images.githubusercontent.com/29139786/81561931-9ebe7d80-93b1-11ea-8a42-69392d586f3d.png">
You can see here

Reply:
I think you haven't fixed the last part of the docstring 
>  list of interactions used in an exploration.